frontend/src/config/config.ts

```diff
@@ -0,0 +1,12 @@
+// import * as fs from 'fs'
+import { Config } from '../constants/datatypes'
+
+export let CONFIG: Config
+
+// tslint:disable:no-var-requires
+CONFIG = require('../../../shared/config.json')
+if (process.env.ENV !== undefined) {
+    CONFIG.title_suffix = (process.env.ENV !== 'PRD' ? ' - ' + process.env.ENV : '')
+}
+
+CONFIG.api.root = process.env.API_ROOT || CONFIG.api.root
```

Comment:
why would we sometimes want to set with env, sometimes with config? seems like it could get confusing

Reply:
We may want to keep this secret

Reply:
Should I add a comment?